docs/sources/administration/configuration.md

```diff
@@ -1426,3 +1426,6 @@ interval_year = YYYY
 Set this to `true` to have date formats be automatically be derived from browser locale. Defaults to `false`. This
 is an experimental feature right now with a few problems that remain unsolved.
 
+### default_timezone
+
+Used as the default timezone for user preferences. Can be either `browser` for the browser local timezone or a timezone name from IANA Time Zone database 
```

Comment:
Would be good with a couple examples here to understand what format is expected.

```suggestion
Used as the default timezone for user preferences. Can be either `browser` for the browser local timezone or a timezone name from IANA Time Zone database, e.g. UTC or Europe/Amsterdam etc.
```